<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientInfoMng</class>
 <widget class="QDialog" name="ClientInfoMng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户信息管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>数据查询</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="2,2,1">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>客户代码</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leqryId"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>开户时间上限</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="de_start">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2015</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空参数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>客户姓名</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leqryName"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>开户时间下限</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="de_end">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2015</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tvIns">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,5,1,2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="btnGoFirst">
           <property name="text">
            <string>首页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGoPre">
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGoNext">
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGoLast">
           <property name="text">
            <string>尾页</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="l_count">
         <property name="text">
          <string>共0页</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,5,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="btnGoNum">
           <property name="text">
            <string>跳转至</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lPageNum"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>页</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
